force-app/main/default/pages/STG_PanelRenameHouseholds.page

```diff
@@ -17,26 +17,67 @@
         }
     </script>
     <apex:form id="form" styleClass="slds-m-around_x-large">
-        <c:STG_PageHeader sectionLabel="{!$Label.stgNavBulkProcesses}" pageLabel="{!$Label.stgHHNameRefreshTitle}" />
+        <c:STG_PageHeader sectionLabel="{!$Label.stgNavBulkProcesses}" pageLabel="{!$Label.stgHHDataRefreshTitle}" />
```

Comment:
This label does not exist in the Custom Label metadata

Reply:
@npsp-reedestockton - Is this a label you need me to write?

Reply:
@sam-knox @andrewyu-salesforce That's text you already provided @sam-knox.  "Refresh Household Data".  In flipping things around with the pages I neglected to add it - it was previously a modification of an existing label.